website/docs/cdktf/concepts/modules.mdx

```diff
@@ -87,6 +87,8 @@ For local modules, use the object format.
 }
 ```
 
+For performance reasons we don't automatically generate bindings for submodules. To generate bindings for submodules, specify the module source as `terraform-aws-modules/vpc/aws//submodules/vpc-endpoints`, where after the `//` is the path to the submodule in the modules repository. Please visit [the terraform source specification](https://www.terraform.io/language/modules/sources) for more details.
```

Comment:
I don't think we need to do a full link here because this is internal to our site! I think we can just do the path after the terraform.io part :-) 

```suggestion
For performance reasons, we don't automatically generate bindings for submodules. To generate bindings for submodules, specify the module source as `terraform-aws-modules/vpc/aws//submodules/vpc-endpoints`, where after the `//` is the path to the submodule in the modules repository. Refer to [the Terraform source specification](/language/modules/sources) for more details.
```